src/content/en/updates/posts/2016/07/infinite-scroller.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: updates/post
+title: "Complexities of an infinite scroller"
+description: "Infinite scrollers are a common UI pattern. We are trying to solve the technical challenges hidden within."
+published_on: 2016-07-01
```

Comment:
This description is very general. Could you write one that mentions specific PWA-related problems?